documentation/Configuration.adoc

```diff
@@ -0,0 +1,18 @@
+== Configuration
```

Comment:
Could you rename this file and "Installation.adoc" to lowercase?

Reply:
i think i made it... anyway i will try to fix it